frontend/src/components/organisms/form-sections/DonneesSection/Scopes.tsx

```diff
@@ -94,13 +94,14 @@ const Scopes = ({
               onChange={
                 triggerWarning && !selectedScopes[value]
                   ? () => {
-                      setWarningType(warningType || 'rgpd');
+                      setWarningType(warningType || ScopeWarningModalType.rgpd);
                       setWarningModalScope(value);
                     }
                   : handleChange
               }
               name={`scopes.${value}`}
-              disabled={disabledApplication || required}
+              disabled={disabled || required}
+              // @ts-ignore
               value={selectedScopes[value]}
```

Comment:
Why not take time to switch `CheckboxInput` component to TypeScript instead of using `@ts-ignore` ?